metaci/cumulusci/models.py

```diff
@@ -50,6 +89,11 @@ def unlock(self):
         if not self.scratch:
             cache.delete(self.lock_id)
 
+    def clean(self):
+        super(Org, self).clean()
```

Comment:
Is the `super` line required? Looks like this method is meant to be overridden anyway https://docs.djangoproject.com/en/1.10/ref/models/instances/#django.db.models.Model.clean

Reply:
you're super right.